.github/dependabot.yml

```
@@ -5,3 +5,8 @@ updates:
   schedule:
     interval: daily
   open-pull-requests-limit: 99
+  groups:
+    inmanta-dev-dependencies:
+      patterns:
+        - "inmanta-dev-dependencies"
+        - "pytest"
```

Shouldn't we also include lxml here? The connection between lxml and inmanta-dev-dependencies is even mentioned in a comment in requirements.txt

Other than that it looks good
```suggestion
        - "pytest"
        - "lxml"
```

---

I'd missed that, thanks!